.env.development

```diff
@@ -32,4 +32,4 @@ FEATURE_BULK_ENROLLMENT=true
 HOTJAR_APP_ID=''
 HOTJAR_VERSION=6
 HOTJAR_DEBUG=''
-FEATURE_FILE_ATTACHMENT=false
+FEATURE_FILE_ATTACHMENT=true
```

Comment:
Are we enabling this feature for production too?

Reply:
not yet, once i test on stage, i will create edx-internal PR to enable it on production.

Reply:
Sounds good